src/model/statsd.coffee

```diff
@@ -0,0 +1,173 @@
+http = require 'http'
```

Comment:
Ok yea, was wondering where that file should go.. am moving it to the api folder.